src/main/resources/org/clulab/wm/eidos/english/grammars/gtri-aske-scientific-modeling.yml

```diff
@@ -0,0 +1,21 @@
+rules:
+
+  - name: gtri-aske-scimod-def1
+    label: Definition
+    priority: 2
+    type: token
+    pattern: |
+      # Simple definition pattern where a Concept is defined after a comma for another Concept
+      @effect:Concept ("," @cause:Concept)
+
+
+  - name: gtri-aske-scimod-def2
+    label: Definition
+    priority: 2
+    example: "Epidemiology is the study and analysis of the distribution (who, when, and where) and determinants of health and disease conditions in defined populations."
+    type: token
+    pattern: |
+      # Rule that recognizes X is the Y (X is a concept and Y is anything else within the sentence.) Needs to be further scoped.
+      @effect:Concept
+      [word="is"] [word="the"]
+      (?<cause> []+ )
```

Comment:
It looks like this matches all the way to the end of the sentence. Can you capture the argument to "is"? 

For example:

Input: "Pandology is the study of pandas, for the purpose of learning" 
Output: "Pandology is the study of pandas"

Reply:
In the case of the sample sentence: "Epidemiology is the study and analysis of the distribution (who, when, and where) and determinants of health and disease conditions in defined populations." I think definition is indeed everything after "is the" to the end of the sentence, for that input, and so this rule covers that.

What you've written is a different rule with different input than the prior rule was design for. The current rule is really broad though and want to scope it, probably including the ability to end the definition capture when any number of linguistic phenomena, that signal to a human, the definition is over. So like in your example. 

However, I think it would be a valid definition to extract that Pandology is the "study of pandas, for the purpose of learning" not just "study of pandas".

I don't believe there is an argument to "is" per se, but sounds like you mean something like an nmod or other set of dependency relationships and we can look into it but I think it's a separate rule or a feature request to scope this "X is the Y" rule.

Reply:
Yeah I don't know whether we want to get the shorter or longer match. Let's roll with this for now, because we can always remove information downstream, but can't reconstitute it.

Reply:
In future work for this one with "X is the Y", it may be possible to further scope the rule to dynamically contract/expand extraction depending on the presence or lack of certain phenomena when present. Phenomena that cues the human about the continuance or conclusion of text representing the definition.